sensor/common/sensor/central_communication_impl.go

```diff
@@ -139,17 +140,14 @@ func (s *centralCommunicationImpl) sendEvents(client central.SensorServiceClient
 		PolicyVersion:            policyversion.CurrentVersion().String(),
 		DeploymentIdentification: configHandler.GetDeploymentIdentification(),
 		SensorState:              s.getSensorState(),
+		RequestDeduperState:      s.clientReconcile,
 	}
+
 	capsSet := set.NewSet[centralsensor.SensorCapability]()
 	for _, component := range s.components {
 		capsSet.AddAll(component.Capabilities()...)
 	}
-	if s.clientReconcile {
-		log.Info("Sensor is capable of doing client reconciliation")
-		capsSet.Add(centralsensor.SensorReconciliationOnReconnect)
-	} else {
-		log.Info("Sensor has client reconciliation disabled")
-	}
+	capsSet.Add(centralsensor.SensorReconciliationOnReconnect)
```

Comment:
Not sure if it would be better to guard this with the FF

Reply:
See comment below